domains/authorization/Authorization.API/API.UnitTests/API.UnitTests/Repository/FakeAffilicationRepository.cs

```diff
@@ -0,0 +1,64 @@
+using API.Authorization.Exceptions;
+using API.Models;
+using API.Repository;
+using MockQueryable.NSubstitute;
+
+namespace API.UnitTests.Repository;
+
+public class FakeAffiliationRepository : IAffiliationRepository
+{
+    private readonly List<Affiliation> _entities = [];
+
+    public Task<Affiliation> GetAsync(object[] keys, CancellationToken cancellationToken)
```

Comment:
I dont entirely understand: object[] keys. Is it a combined key?

Reply:
Removed repo